shop/src/app/components/presentational/products-list-view/products-list-view.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { ProductDetails } from 'src/app/modules/shared/types/product-details.types';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router';
+import { getProductDetailsMock } from 'src/app/mocks/products.mocks';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-products-list-view',
+  templateUrl: './products-list-view.component.html',
+  styleUrls: ['./products-list-view.component.scss']
+})
+
+export class ProductsListViewComponent {
+  @Input() productList: ProductDetails[] | undefined;
+
+  products: Observable<ProductDetails>;
```

Comment:
You are expecting an `Observable` here, but in the ProductsMock you are returing plain data. Can you please adapt this accordingly? 
Either change the variable definition or adapt the mocking service.
For more details regarding Observables & Subcriptions: https://www.telerik.com/blogs/angular-basics-introduction-observables-rxjs-part-2

Reply:
I have since changed from mocks to the actual backend, and it is now an Observable of the type that I am expecting.